infrastructure/ci/dev-platform/shared/create-slack-stack.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eu
+
+echo Creating stack for Slack Notification
+
+if [[ "$1" == "" ]]; then
+  echo "Usage: $0 <development | build | staging | integration | production>"
```

Comment:
This is not the actual usage instructions; it should detail the other two parameters also.

Reply:
fixed